source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -0,0 +1,407 @@
+
+   See :ref:`Tapping Logic <numark-platinum-fx-tapping-logic>`"
+   "", "Left :hwlabel:`PFL` ON (Right off)", "Set BPM for left deck.
+
+   See :ref:`Tapping Logic <numark-platinum-fx-tapping-logic>`"
+   "", "Right :hwlabel:`PFL` ON (Left off)", "Set BPM for right deck.
+
+   See :ref:`Tapping Logic <numark-platinum-fx-tapping-logic>`"
+   "", ":hwlabel:`SHIFT` Default Tempo Reset", "Press Shift and this button to reset the tempo to the track to default BPM.
+
+   Note: Reset the selected deck. See :ref:`Tapping Logic <numark-platinum-fx-tapping-logic>`"
+   "24", "Platter/Jog Wheel", "This capacitive, touch-sensitive jog wheel controls the audio when the wheel is touched and moved. When the Scratch button is not active, use the jog wheel to bend the pitch of the track. When the Scratch button is active, use the jog wheel to grab and move the audio, scratching the track as you would with a vinyl record. You can also grab the non-touch-sensitive outer wheel, to bend the pitch of the track."
+   "24", ":hwlabel:`SHIFT` Scrub Scroll", "Scrub Scrolling while holding Jog wheel."
+   "", "Deck Display", "**1. Active Deck:** Indicates the currently active deck.
+
+   **2. Platter Position:** Displays the current track position.
+
+   **3. BPM:** Displays the current BPM for the selected track.
+
+   **4. Keylock:** This icon illuminates when keylock is active for the current deck.
+
+   **5. Pitch Range:** Shows the current pitch range.
+
+   **6. Pitch:** Shows current pitch of the track.
+
+   **7. Pitch Adjust:** Indicates the direction to move Pitch Fader to match the current track's BPM with the track on the opposite deck.
+
+   **8. Time Remaining:** Displays the time elapsed or time remaining for the current track. (Time elapsed only implemented in current mapping).
+
+   **9. Time Bars:** Provides visual reference to time remaining on the selected song. When the song is almost over these segments will flash to give a warning."
+   "25", "Scratch", "Press this button to turn on the scratch feature for the jog wheel."
+   "", ":hwlabel:`SHIFT` Deck Select", "Switch between decks 1 & 3 or 2 & 4 for 4 deck mixing"
+   "26", "Shift", "Allows multiple control commands to be triggered when pressed first along with other buttons."
+   "27", "Sync", "**Tap** this button to automatically match the corresponding Deck's tempo with the opposite Deck's tempo."
+   "", ":hwlabel:`HOLD` Lock Sync", "Hold for at least one second to lock sync. Tap again to release sync lock."
+   "", ":hwlabel:`SHIFT` Quantize", "Toggle quantize on/off."
+   "28", "Cue", "Sets and recalls the main Cue point in the current track"
+   "", ":hwlabel:`HOLD` Cue Play", "Hold down the Cue button for temporary play of the cue point. The track will play for as long as the button is held down and return to the cue point once it is released"
```

Comment:
```suggestion
   "28", "Cue (tap)", "Sets and recalls the main Cue point in the current track"
   "", "Cue Play (hold)", "Play from the Cue point for as long as the button is held down and return to the cue point once it is released"
```

Reply:
actually this is standard cue behaviour (CueButton component uses [`cue-default`](https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[ChannelN]-cue_default))